build.py

```diff
@@ -785,7 +785,7 @@ def configure(conf, env):
     try:
         print "Checking for updates to ninja module..."
         subprocess.check_call(['git', '-C', my_dir, 'fetch'])
-        output = subprocess.check_output(['git', '-C', my_dir, 'log', '--oneline', '@..origin'])
+        output = subprocess.check_output(['git', '-C', my_dir, 'log', '--oneline', '@..@{push}'])
```

Comment:
I think you want @{upstream} since most people may not set this up to push. And if they do, it may be different than where they would pull from (eg push to personal fork).